collaboration/sig-core/proposals/test-execution-using-labels.md

```diff
@@ -0,0 +1,93 @@
+# Test execution organization using labels
+
+Created 2020-03-16 by Korbinian Stoemmer (@k15r)
+
+## Motivation
+
+Kyma developers write a lot of tests. These tests have different intentions and thus require different conditions to run successfully. In order to run those tests under the expected conditions on our CI Infrastructure, multiple shell scripts have been implemented. 
+These shell scripts are tailored to the exact implementation of the test at the time of writing. This makes it really hard to reimplement a test, or to add a new test that requires the same setup (conditions) as another test.
+
+Example:
+
+currently the backup and restore tests are executed the following way on our CI:
+* install kyma
+* install `TestDefinition` with parameter indicating before backup test
+* find all `TestDefinitions` with specific app label
+* create `ClusterTestSuite` referencing the found `TestDefinitions` and wait for success
+* take backup
+* spinup new cluster and restore
+* install `Testdefinition` with parameter indicating after restore test
+* create `ClusterTestSuite` referencing the found `TestDefinitions` and wait for success
```

Comment:
```suggestion
1. Install Kyma.
2. Install `TestDefinition` with parameter indicating before backup test.
3. Find all `TestDefinitions` with specific app label.
4. Create `ClusterTestSuite` referencing the found `TestDefinitions` and wait for success.
5. Take backup.
6. Spin up new cluster and restore.
7. Install `TestDefinition` with parameter indicating after restore test.
8. Create `ClusterTestSuite` referencing the found `TestDefinitions` and wait for success.
```

As per [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/04-formatting.md#ordered-and-unordered-lists), we use ordered lists if the order of the performed steps matter. They also state to capitalize the first letter of a bullet point, and end it with a period if it is a sentence.